VideoApi/VideoApi.Common/Security/MagicLinksJwtTokenProvider.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using VideoApi.Common.Configuration;
+using VideoApi.Domain.Enums;
+
+namespace VideoApi.Common.Security
+{
+    public class MagicLinksJwtTokenProvider : IMagicLinksJwtTokenProvider
+    {
+        private readonly MagicLinksConfiguration _magicLinksConfiguration;
+        public const int ExpiryTimeInMinutes = 60 * 8;
```

Comment:
is this used in the tests file, if not perhaps make it private, otherwise it might give sonarlint warnings.